src/js/browser-plugin.js

```diff
@@ -974,6 +1098,64 @@ function openClaimPopup(tweet) {
 
 }
 
+function openNotTaggedFeedbackPopup(tweet) {
+  
+  const elementTxt = browserAPI.i18n.getMessage('tweet_post');
+
+  let popupTitle = browserAPI.i18n.getMessage('not_tagged', elementTxt);
+  let popupButtonText = browserAPI.i18n.getMessage('ok');
+
+  //let meterLogoSrc = browserAPI.extension.getURL(imgsPath + "meter.png");
+  let meterLogoSrc = null;
+  
+  return Swal2.fire({
+    type: 'warning',
+    title: popupTitle,
+    showCloseButton: true,
+    showCancelButton: false,
+    confirmButtonColor: buttonColor,
+    confirmButtonText: popupButtonText,
+    html:
+      '<span>' + browserAPI.i18n.getMessage('wait_label_for_feedback', elementTxt) + '</span>',
+    footer:
+      `<img class="coinformPopupLogo" src="${minlogoURL}"/>` +
+      '<span>' + browserAPI.i18n.getMessage('popup_footer_text') + '</span>',
+    focusConfirm: true,
+  }).then(function (result) {
+    if(result.value === true){
+      // function when confirm button is clicked
+    }
+  });
+
+}
+
+function openNotLoggedFeedbackPopup(tweet) {
+
+  let popupTitle = browserAPI.i18n.getMessage('not_logged');
+  let popupButtonText = browserAPI.i18n.getMessage('ok');
+  
+  return Swal2.fire({
+    type: 'warning',
+    title: popupTitle,
+    showCloseButton: true,
+    showCancelButton: false,
+    confirmButtonColor: buttonColor,
+    confirmButtonText: popupButtonText,
+    html:
+      '<span>' + browserAPI.i18n.getMessage('provide_feedback_with_login') + '</span><br/>' +
+      '<span>' + browserAPI.i18n.getMessage('login_register_instructions') + '</span>',
+    footer:
+      `<img class="coinformPopupLogo" src="${minlogoURL}"/>` +
+      '<span>' + browserAPI.i18n.getMessage('popup_footer_text') + '</span>',
+    focusConfirm: true,
+  }).then(function (result) {
+    if(result.value === true){
+      // function when confirm button is clicked
```

Comment:
sam as above

Reply:
Again its the confirm button for another informative message (swal2) and it will be removed too.